src/main/cpp/subsystems/Climber.cpp

```diff
@@ -0,0 +1,30 @@
+#include "subsystems/Climber.h"
+
+Climber::Climber() 
+: m_pid_controller(0.1, 0.0, 0.0)
+, m_motor(5, rev::spark::SparkLowLevel::MotorType::kBrushless)
+, m_climber_up(false)
+{
+    frc2::CommandScheduler::GetInstance().RegisterSubsystem(this);
+}
+
+frc2::CommandPtr Climber::FlipArm()
+{
+    return this->RunEnd(
+        [this] {
+            if (m_climber_up){
+                m_motor.Set(m_pid_controller.Calculate(m_motor.GetEncoder().GetPosition(), 0.5));
+
+            }
+            else
+            {
+                m_motor.Set(m_pid_controller.Calculate(m_motor.GetEncoder().GetPosition(), 33.0));
+            }
+        },
+        [this]
+        {
+            m_climber_up = !m_climber_up;
+            m_motor.Set(0.0);
```

Comment:
`SparkMax` (well, technically `SparkBase`) has a `StopMotor()` method:

```suggestion
            m_climber_up = !m_climber_up;
            m_motor.StopMotor();
```

Does that read better to you? If not, feel free to reject.

Reply:
changed to StopMotor